flight/src/tests.rs

```diff
@@ -0,0 +1,100 @@
+use super::*;
+use crate::types::{ConclusionData, ConclusionEvent, ConclusionInit};
+use arrow::array::{Array, BinaryArray, ListArray, StringArray, UInt8Array};
+use ceramic_core::StreamIdType;
+use cid::Cid;
+
+/// Tests the conversion of ConclusionEvents to Arrow RecordBatch.
+///
+/// This test creates mock ConclusionEvents (both Data and Time events),
+/// converts them to a RecordBatch using the conclusion_events_to_record_batch function,
+/// and then verifies that the resulting RecordBatch contains the expected data.
+///
+/// The test checks:
+/// 1. The number of rows in the RecordBatch
+/// 2. The schema of the RecordBatch
+/// 3. The content of each column in the RecordBatch
+#[test]
+fn test_conclusion_events_to_record_batch() {
+    // Create mock ConclusionEvents
+    let events = vec![
+        ConclusionEvent::Data(ConclusionData {
+            id: Cid::default(),
+            init: ConclusionInit {
+                stream_type: StreamIdType::Model as u8,
+                controller: "did:key:test1".to_string(),
+                dimensions: vec![],
+            },
+            previous: vec![],
+            data: vec![1, 2, 3],
+        }),
+        ConclusionEvent::Data(ConclusionData {
+            id: Cid::default(),
+            init: ConclusionInit {
+                stream_type: StreamIdType::Model as u8,
+                controller: "did:key:test2".to_string(),
+                dimensions: vec![],
+            },
+            previous: vec![Cid::default()],
+            data: vec![4, 5, 6],
+        }),
+    ];
+    // Convert events to RecordBatch
+    let record_batch = conclusion_events_to_record_batch(&events).unwrap();
+
+    // Assert the number of rows
+    assert_eq!(record_batch.num_rows(), 2);
+
+    // Assert the schema
+    let schema = record_batch.schema();
```

Comment:
Lets validate these tests using the `expect_test` pattern. This way they are much easier to audit and update.

You can convert a RecordBatch into string like this https://github.com/ceramicnetwork/rust-ceramic/pull/500/files#diff-eb701a6fc846dc0a4b096209cdf1b5b1a75028561fcc2e3c6629862757eec348R287

Then you can use `expect!` like this to validate its what we expect https://github.com/ceramicnetwork/rust-ceramic/pull/500/files#diff-eb701a6fc846dc0a4b096209cdf1b5b1a75028561fcc2e3c6629862757eec348R310-R315

If you haven't used expect_test before I recommend a quick read on how to use it and update tests https://docs.rs/expect-test/latest/expect_test/#introduction